src/components/buttons/add/AddLandmarkModal.js

```diff
@@ -0,0 +1,143 @@
+import React, {useState} from "react";
+import Modal from "react-bootstrap/Modal";
+import Button from "@mui/material/Button";
+import {Input} from "reactstrap";
+
+const AddLandmarkModal = ({
+                              show,
+                              onHide,
+                              addLandmark,
+                              onChangeIsAddedNewLandmark,
+                              name,
+                              location,
+                              textDescription,
+                              category,
+                              price,
+                              nameInputChangeHandle,
+                              locationInputChangeHandle,
+                              textDescriptionInputChangeHandle,
+                              categoryInputChangeHandle,
+                              priceInputChangeHandle
+                          }) => {
+    const [requiredMessage, setRequiredMessage] = useState('');
+
+    const handleSave = () => {
+        if (!name || !location || !textDescription || !category || price == 0) {
+            setRequiredMessage('Complete all fields!')
+            return;
+        }
+        setRequiredMessage('');
+        addLandmark();
+        onChangeIsAddedNewLandmark((prev) => !prev);
+        onHide();
+    }
+
+    const handleClose = () => {
+        setRequiredMessage('');
+        nameInputChangeHandle('')
+        locationInputChangeHandle('')
+        textDescriptionInputChangeHandle('');
+        categoryInputChangeHandle('');
+        priceInputChangeHandle(0);
+        onHide();
+    }
+
+    return (
+        <Modal
+            show={show}
+            size="xs"
+            aria-labelledby="contained-modal-title-vcenter"
+            centered
+        >
+            <Modal.Header>
+                <div style={{
+                    backgroundColor: "black",
+                    padding: "5px",
+                    textAlign: "center",
+                    marginLeft: "8rem",
+                    color: "white"
+                }}>
+                    <Modal.Title id="contained-modal-title-vcenter">
+                        Add a landmark
+                    </Modal.Title>
+                </div>
+            </Modal.Header>
+            <Modal.Body>
```

Comment:
You omitted to add the input fields for image and audio

Reply:
I didn't put them because on backend the image and audio files are not saved properly, so until it's done on backend side it's useless.

Reply:
There is an endpoint that implements the functionality of adding a tourist objective. You will need to send the image and the audio file in the form of strings of bytes.

Reply:
That endpoint is not working. If you want to save also the image or audio file for a landmark as form-data via postman is not working.

Reply:
If you send an array of bits, it works. If you send the image as a file, it is expected that it will not work.

Reply:
I tested the endpoint that offers the functionality of adding landmarks and it works. For the test, I used both postman (I chose to use JSON as the transfer mode), as well as using the existing React application within the project and the data was successfully saved in the database.